frontend/src/Components.res

```diff
@@ -934,7 +934,24 @@ module Link = {
         []
       },
     })
-    <a className={Sx.make(sx)} ?href ?target> icon_svg text </a>
+    <a
+      className={Sx.make(sx)}
+      href
+      ?target
+      onClick={event =>
+        // https://github.com/MinimaHQ/safe-routing-blog-app-example/blob/master/02-multiple-types/src/Router.re
+        if (
+          !(event->ReactEvent.Mouse.defaultPrevented) &&
+          (event->ReactEvent.Mouse.button == 0 &&
+          (!(event->ReactEvent.Mouse.altKey) &&
+          (!(event->ReactEvent.Mouse.ctrlKey) &&
+          (!(event->ReactEvent.Mouse.metaKey) && !(event->ReactEvent.Mouse.shiftKey)))))
```

Comment:
react-router uses the same logic: https://github.com/ReactTraining/react-router/blob/346153e38365e6a88657ce2e4e5acada290fc430/packages/react-router-dom/modules/Link.js#L17